applications/Unity.GrantManager/src/Unity.GrantManager.Application/Intake/SubmissionAppService.cs.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Extensions.Configuration;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq.Dynamic.Core;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Volo.Abp.Application.Dtos;
+
+namespace Unity.GrantManager.Intake;
+
+public class SubmissionAppService : GrantManagerAppService, ISubmissionAppService
+{
+    private readonly RestClient _intakeClient;
+    private static List<string> _summaryFieldsFilter
+    {
+        // NOTE: This will be replaced by a customizable filter.
+        get
+        {
+            return new List<string>(new string[] {
+                "projectTitle",
+                "projectLocation",
+                "contactName",
+                "organizationLegalName",
+                "eligibleCost",
+                "totalRequestToMjf"
+            });
+        }
+    }
+
+    public SubmissionAppService(RestClient restClient)
+    {
+        _intakeClient = restClient;
+    }
+
+    public async Task<object> GetSubmission(Guid? formSubmissionId)
+    {
+        if (formSubmissionId == null)
+            throw new ApiException(400, "Missing required parameter 'formId' when calling GetSubmission");
+
+        var request = new RestRequest($"/submissions/{formSubmissionId}");
```

Comment:
I would consider abstracting this away to an infrastructure layer: maybe that is just a TODO in this case
https://learn.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/ddd-oriented-microservice#the-infrastructure-layer